build.xml

```diff
@@ -70,7 +69,7 @@
         </exec>
         <exec executable="${path.ecs.executable}" logoutput="true" passthru="true" checkreturn="true">
             <arg value="check" />
```

Comment:
The flag `--verbose` is not used here, but it is used in other build definitions. Can you explain or fix it?

Reply:
verbose was added